deployment/docker-compose/docker-compose.yml

```diff
@@ -80,7 +82,7 @@ services:
       dockerfile: Dockerfile
     restart: always
     env_file:
-      - .env
+      - .core_backend.env
```

Comment:
I know this section will have to change according to the embeddings container merge!

Reply:
Updated!